### ISSUE TITLE
Install Repaired but Can't Launch JVM

### ISSUE THREAD

**Issue by aliengeo** [DraqueT/PolyGlot#1183]
I'm on Windows 10, x64 system. This is a new install, not an upgrade.

The Windows installer gave me the "Failed to launch JVM" error, so I tried:
- running the repair tool
- uninstalling through the wizard and reinstalling
- running the .exe directly from program files
- deleting the program directory and repairing
- installing the beta on top of the stable version
- repairing the beta
- deleting temporary files
- restarting my computer
- deleting both versions through the system Add or Remove Programs menu and reinstalling
- installing the stable version on top of the beta

Unfortunately, none of this worked. Is there anything else I can try?

**Comment by DraqueT** [comment]
Thank you for opening a ticket and for the thoroughness of your testing/documentation here!

I'm curious whether this failure is happening on JFX initialization. After you try to open PolyGlot, could you please check in ~/PolyGlot for an PolyGlot_error.log file? If it's there could you please attach it here for me to take a look at? If it's not there at all let me know as well, as that means it's not even getting into the boot phase.

**Comment by aliengeo** [comment]
Just checked—I couldn't find any log file in `Program Files/Polyglot`.

**Comment by DraqueT** [comment]
Ok, try this:

1) Navigate to C:\Program Files\PolyGlot (or wherever you installed PolyGlot)
2) Right click in folder -> Open in Terminal
3) enter PolyGlot.exe and hit enter
4) Wait for it to crash
5) Copy/paste the result here

Ultimately there's something different on your system from the test systems that I am running. It might be some edge case that I haven't thought of, but hopefully this will shed some light on it.

**Comment by aliengeo** [comment]
(For reference if you want to write this up for troubleshooting purposes: I first had to install Windows Terminal and restart my computer to see that option in the context menu; also while `PolyGlot.exe` did run in Command Prompt, the command that worked in PowerShell was `.\PolyGlot.exe`)

In PowerShell:

> `PS C:\Program Files\PolyGlot> .\PolyGlot.exe`
> `PS C:\Program Files\PolyGlot>`

In an admin Command Prompt:

> `C:\Program Files\PolyGlot>polyglot.exe`
> `C:\Program Files\PolyGlot>`

In both cases, it opened the JVM error dialog as before, with no output to terminal.

**Comment by DraqueT** [comment]
Thanks for the added details there. I am sure that you have already tried this, but have you done an uninstall of PolyGlot, made certain the files in ProgramFiles are removed, then reinstalled? This definitely does not have to do with PolyGlot's temporary files, as they're only read once the JVM starts up and the actual code of PolyGlot begins executing. There's got to be some bizarre state that the JVM itself is getting into.

**Comment by aliengeo** [comment]
Yeah, I did a double check just now and it seems to uninstall from Program Files correctly. Didn't leave a folder or a broken desktop shortcut.

It did leave the empty folder `Users\[user]\PolyGlot`, and rebuilt it on next run if I deleted it, but I assume that's where individual conlang data is stored?
